demystifying_large_language_models/demystifying_large_language_models.md

```diff
@@ -0,0 +1,691 @@
+
+## LLM Costs and Risks
+
+Costs
+---
+
+Over time, with advances in the speed and size of computer processors (especially Graphical Processing Units, or GPUs), faster and more efficient data storage, and competition among cloud providers, computing has become less expensive.  That means that tasks can be performed today that would have been unrealistic (in terms of time and expense) to perform a few years ago.  That's true across computing, and it's certainly true in artificial intelligence.  But "less expensive" doesn't mean free.  Large language models are incredibly resource intensive, in terms of the amount of input required, the amount of computing required, and the energy consumed in building and running the models. 
+
+
+Risks
+---
+
+Additionally, there are important risks to consider.  We're going to borrow a framework from Weidinger et al.  In their superb and approachable ["Ethical and social risks of harm from Language Models,"](https://arxiv.org/abs/2112.04359) Weidinger and her co-authors enumerate six domains of risks, harms, and costs associated with large language models.  We summarize these risks below (and note, we have avoided the temptation to ask a large language model to summarize the text for us):
+
+1. Discrimination, exclusion, and toxicity
+-----
+
+Where can one obtain huge amounts of text, generated by humans, for free?  The internet!  Language models and their abilities to predict text have grown in part because of better, faster, and cheaper computers, but also in part because of the massive proliferation of human-generated text that can be analyzed.  Unfortunately, this set of language is biased in terms of who generates it and the opinions they share.  Toxic language, stereotypes, and bias abound, and these have already been documented in text generated by LLMs.  
+
+2. Information Hazards
+------
+
+Because the training data given to LLMs is so vast, there are cases (some involving litigation) of private information such as home addresses being leaked by LLMs.  Additionally, LLMs might correctly infer and disclose sensitive information while amassing and summarizing data (for example, accurately disclosing a serious disease diagnosis to an unprepared user asking about their symptoms).
+
+While the authors do not explicitly mention this, another information hazard we want to point out is the risk related to input text.  Users can disclose private or regulated information to an LLM via prompts.  One of the ways that LLMs acquire additional language samples is by adding the prompts that users supply to their vast collection of language examples.  Supplying private information (about human subjects, about companies, and so on) when asking LLMs to generate text could constitute an impermissible breach of privacy or violation of intellectual property law or other regulation.
```

Comment:
This doesn't match my understanding of what an information hazard is, at least as originally defined by Bostrom. An information hazard is the hazard that arises from someone having access to information that can allow them to cause harm to someone else, or causes harm to them directly (e.g., harm of finding out you have a disease while being unprepared, not necessarily having access to supports, etc)  

This example seems to moreso be something that is upstream of an information hazard -- that is, it's an action that could add to model training in a way that _creates future information hazards_, but is not an information hazard itself.

Reply:
I'm taking this directly from the cited article and accurately represents Weidinger et al., so let's leave it be, and allow the primary authors to duke terminology out amongst themselves.

Reply:
Oops, looks like my comment may have been on too big of a section of the module -- I'm referring specifically only to the paragraph that starts with "While the authors do not explicitly mention this," 

If it is the case that you think this is in line with what the authors are saying, it might be worth rewriting the start of the sentence then, as I took this as your own riff on things, and not that the authors of the article called providing private information as input text an information hazard itself. (This is in line with my own reading of that section of the article, but I didn't read the entire article itself). 

I also have a lot of friends working in the AI safety space (of which Bostrom's "Superintelligence" is a seminal text) who do a lot of thinking + talking about info hazards -- could easily get a quick look over from them (by end of day) to see if they think this is the right place to frame this.

Reply:
OH OH OH gotcha you're right, can you make a stab at a suggestion for a change?  Maybe "information-related" risk?  IDK